common/servertalk.h

```diff
@@ -277,6 +277,7 @@
 #define ServerOP_ReloadLoot 0x4127
 #define ServerOP_ReloadBaseData 0x4128
 #define ServerOP_ReloadSkillCaps 0x4129
+#define ServerOP_ReloadNpcSpells 0x4130
```

Comment:
Can we make this `ServerOP_ReloadNPCSpells` just so it's similar to the other `ServerOP_ReloadNPCEmotes` naming?